public/app/plugins/datasource/grafana-azure-monitor-datasource/types/index.ts

```diff
@@ -83,19 +89,19 @@ export interface AzureDataSourceJsonData extends DataSourceJsonData {
   subscriptionId?: string;
 
   // logs
+  logAnalyticsDefaultWorkspace?: string;
+  /** @deprecated Azure Logs credentials */
   azureLogAnalyticsSameAs?: boolean;
   logAnalyticsTenantId?: string;
   logAnalyticsClientId?: string;
   logAnalyticsSubscriptionId?: string;
```

Comment:
should we add the @deprecated comment to all four properties so we get proper deprecation messages in editors?